app/lib/import_export_constants.rb

```diff
@@ -262,6 +264,7 @@ module ImportExportConstants # rubocop:todo Metrics/ModuleLength
     updated_at: 'Monitoree Updated Date',
     # Monitoring Info - Linelist Info
     workflow: 'Current Workflow',
+    enrolled_workflow: 'Enrolled Workflow',
```

Comment:
Should these new fields (`enrolled_workflow` and `contact_became_case_at`) be supported in the custom export? I'm not seeing them there.

Reply:
question for @hackrm but probably